monkey/infection_monkey/master/control_channel.py

```diff
@@ -37,10 +38,9 @@ def should_agent_stop(self) -> bool:
             response = json.loads(response.content.decode())
             return response["stop_agent"]
         except Exception as e:
```

Comment:
For all of the exceptions here, we should be a little more specific than catching `Exception`. If something were to raise a `ValueError`, that would probably indicate something has gone horribly wrong and we should shutdown. Determine what kinds of exceptions `requests` will throw and which of those we're willing to handle.